conda_mirror/conda_mirror.py

```diff
@@ -215,12 +215,12 @@ def cli():
     if args.version:
         from . import __version__
         print(__version__)
-        return
+        sys.exit(1)
```

Comment:
I always like the pattern of returning the exit code and then doing the sys.exit() in the main. That way there are no sudden exits playing hide-and-seek in the code when you're trying to, for example, test.